supervisor/tools/testrun_analyzer.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/python3
```

Comment:
`supervisor/tools/testrun_analyzer.py` would be better as `supervisor/ewa_utils.py` (I'd call it "ewa_utils" not "tcms_utils" since it's basic function is specifically to analyze EWA-created TCMS testruns, not do things generally with TCMS). It would also be possible to just make it part of `analyze_ew_testrun.py`, but let's keep it separate.